pkg/machinery/resources/network/route_spec_test.go

```diff
@@ -42,7 +42,7 @@ dst: 192.168.3.4/25
 src: 1.1.1.1
 gateway: 2.2.2.2
 outLinkName: eth0
-table: local
```

Comment:
these changes are wrong as well

Reply:
How would you like to approach the situation, routing tables are integers not strings.  Should the definition treat this as strings and convert it internally as integers?

Reply:
just keep things as they are, and if the change is made, it should be backwards compatible